sources/packages/backend/apps/queue-consumers/test/helpers/testing-modules/testing-modules-helper.ts

```diff
@@ -44,7 +45,7 @@ export async function createTestingAppModule(): Promise<CreateTestingModuleResul
   );
   const sshClientMock = createMock<Client>();
   const module: TestingModule = await Test.createTestingModule({
-    imports: [QueueConsumersModule],
+    imports: [QueueConsumersModule, DiscoveryModule],
```

Comment:
@dheepak-aot FMU, can you please briefly tell why the `DiscoveryModule` was imported here.

Reply:
Discovery module is something that we have been using to get the provider/context from a particular module. 

When we try to get a provider from a nestjs application using app.get(), it does not guarantee if the instance of the provider was returned  from root module, or one of the child modules of the root module.

So we use the discovery service(which require the discovery module) to get a provider instance form a particular module.

Reply:
Thanks @dheepak-aot . 👍 Nicely explained. 😄